jwql/utils/preview_image.py

```diff
@@ -212,13 +212,6 @@ def get_data(self, filename, ext):
                     yd, xd = data.shape[-2:]
                     dq = np.ones((yd, xd), dtype="bool")
 
-                # Collect information on aperture location within the
```

Comment:
@bhilbert4 This was crashing on stage 3 data products, as some of them don't have `SUBSTRT*` or `SUBSIZE*` in their headers.  But upon further investigation, these attributes didn't seem to be used anywhere?  So I just removed them altogether.  Do you know if these are needed?

Reply:
@bhilbert4 Oh I see now, this actually is used for the creation of mosaics.  I should be able to get around this with some sort of parameter passing and/or `try/except`